java-pubsub-authenticated-push/src/main/java/com/example/appengine/pubsub/PubSubPush.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.appengine.pubsub;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleIdToken;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.xml.bind.DatatypeConverter;
+import org.apache.http.HttpStatus;
+
+@WebServlet(name = "PubSubPush", value = "/_ah/push-handlers/receive_messages")
+public class PubSubPush extends HttpServlet {
+  Data data;
+
+  @Override
+  public void doPost(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+
+    if (null == req.getParameter("token")
+        || null == req.getHeader("Authorization")
+        || !System.getenv("PUBSUB_VERIFICATION_TOKEN").equals(req.getParameter("token"))) {
+      resp.sendError(HttpStatus.SC_BAD_REQUEST, "Invalid request");
+    }
+
+    String headers[] = req.getHeader("Authorization").split(" ");
```

Comment:
`split(" ");` It would be better to have constant for this white spcae

Reply:
only one place it is used so no need to crate constant